cms/static/js/certificates/models/certificate.js

```diff
@@ -0,0 +1,71 @@
+// Backbone.js Application Model: Certificate
+
+define([
+    'backbone', 'js/certificates/models/signatory', 'js/certificates/collections/signatories', 'underscore', 'underscore.string', 'gettext', 'backbone-relational', 'backbone.associations', 'coffee/src/main'
+],
+function(Backbone, Signatory, SignatoryCollection, _, str, gettext) {
+    'use strict';
+    _.str = str;
```

Comment:
Yes I see that, but it looks like it is imported as underscore.string anyway - is that a separate js library? then it makes sense. It's not a criticism, more a desire to know.